# exception causing motion execution to fail
string execution_exception
